sklearn/preprocessing/data.py

```diff
@@ -930,7 +930,8 @@ def _check_array(self, X, copy):
         return X
 
     def fit(self, X, y=None):
-        """Compute the median and quantiles to be used for scaling.
+        """Compute the median and quantiles to be used for scaling. Note that
```

Comment:
I think this is only true if `with_centering=True`

To respect docstring conventions, you want something like:

```
"""Compute the median and quantiles to be used for scaling.

   Note that RobustScaler cannot be fitted to sparse inputs if with_centering=True.
"""
```